charms/kfp-api/tox.ini

```diff
@@ -33,6 +33,8 @@ allowlist_externals =
     xargs
 commands = 
 ; uses 'bash -c' because piping didn't work in regular tox commands
+  pip-compile requirements.in
+  pip-compile requirements-fmt.in
```

Comment:
```suggestion
```

Reply:
We do this in other places too, e.g:
* https://github.com/canonical/notebook-operators/blob/a8f0aefcc16e9242742c329c08ea16bfe4b8733c/charms/jupyter-controller/tox.ini#L36-L38
* https://github.com/canonical/oidc-gatekeeper-operator/blob/8f98b252c69d32aa3b7fd1b8ce9fbd042d49cd79/tox.ini#L36-L38